src/ProcessManager/ChromeManager.php

```diff
@@ -83,6 +92,19 @@ private function findChromeDriverBinary(): string
         }
     }
 
+    private function getChromeDriverArguments(): array
+    {
+        $args = ['--port='.$this->options['port']];
+
+        // Add custom arguments with PANTHER_CHROME_DRIVER_ARGUMENTS
+        if ($_SERVER['PANTHER_CHROME_DRIVER_ARGUMENTS'] ?? false) {
+            $arguments = explode(' ', $_SERVER['PANTHER_CHROME_DRIVER_ARGUMENTS']);
```

Comment:
It may not work for complex arguments (values containing a space between quotes for instance). Shouldn't we find something smarter here?